src/Crawler.php

```diff
@@ -280,7 +280,7 @@ protected function addAllLinksToCrawlQueue(string $html, Url $foundOnUrl)
             ->each(function (Url $url) use ($foundOnUrl) {
                 $node = $this->addtoDepthTree($this->depthTree, (string) $url, $foundOnUrl);
 
-                if (! $this->shouldCrawlAtDepth($node->getDepth())) {
+                if (! $this->shouldCrawlAtDepth($node)) {
```

Comment:
Rename this method to `shouldCrawl`. That'll better describe it's new purpose.

Reply:
Thanks, I forgot to rename it. 
Done.